### PR TITLE
Hyperlink support for `flutter doctor` in flutter console (#94).

### PR DESCRIPTION
**Pull request by pq**
Fixes: https://github.com/flutter/flutter-intellij/issues/94

/cc @stevemessick @devoncarew

### CODE REVIEW

**Comment by stevemessick** [comment]
LGTM

**Comment by pq** [comment]
Thanks!

File: src/io/flutter/console/FlutterConsoleFilter.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+/*
```
Comment (devoncarew) [inline src/io/flutter/console/FlutterConsoleFilter.java]: This is a new file; we should use the chromium copyright.

Comment (pq) [inline src/io/flutter/console/FlutterConsoleFilter.java]: Whoops!

Hmmm, looks like my settings got borked...

**Pull request merged**